meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-kernel/linux/linux-lmp-ti-staging_git.bb

```diff
@@ -2,14 +2,23 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 include recipes-kernel/linux/kmeta-linux-lmp-6.1.y.inc
 
+KERNEL_REPO ?= "git://git.ti.com/ti-linux-kernel/ti-linux-kernel.git"
+KERNEL_REPO_PROTOCOL ?= "https"
+KERNEL_BRANCH = "ti-linux-6.1.y"
```

Comment:
All the other are assigned with a weak default so It would be nice to have this too

Reply:
Thanks, updated.